src/hooks/useConnectConfig.ts

```diff
@@ -0,0 +1,7 @@
+import connectConfig from 'connect-config';
+import { useState } from 'react';
+
+export function useConnectConfig() {
+    const config = useState(connectConfig);
```

Comment:
We should use `useAtom` instead, so that if the config is changed in one component then the others also get that update

Reply:
np!
replaced here: [a600091](https://github.com/powerhouse-inc/document-model-electron/pull/185/commits/a60009131ad8600174311b45a696d0c3219ae8ce)